manifests/config.pp

```diff
@@ -1,8 +1,13 @@
 # manage a config snippet
 define nftables::config (
+  Pattern[/^\w+-\w+$/] $tablespec = $title,
   Optional[String] $content = undef,
   Optional[Variant[String,Array[String,1]]] $source = undef,
 ) {
+  if $content and $source {
+    fail('Please pass only $content or $source, not both.')
```

Comment:
this does not check the case, when none is passed.

Reply:
Uh, sorry, I'm not sure what you mean. Passing neither is fine, that's the main goal of the change. To allow just declaring tables like this:

```puppet
nftables::config{'ip-fail2ban':}
```

but guess that's not what you mean. Could you please clarify?

Reply:
ahhh, right :p well I already forgot about that when I reviewed this.